packages/xchain-cosmos/src/cosmos/types.ts

```diff
@@ -30,6 +30,18 @@ export type TransferParams = {
   fee?: StdTxFee
 }
 
+export type TransferOfflineParams = {
+  privkey: PrivKey
+  from: string
+  from_account_number: string
+  from_sequence: string
+  to: string
+  amount: BigSource
+  asset: string
+  memo?: string
+  fee?: StdTxFee
+}
```

Comment:
Since it's looks like an extension of `TransferParams` , how about the following?

```ts
export type TransferOfflineParams = TransferParams & {
  from_account_number: string
  from_sequence: string
}
```